features/dd-trace-core/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import com.datadog.gradle.config.androidLibraryConfig
+import com.datadog.gradle.config.dependencyUpdateConfig
+import com.datadog.gradle.config.javadocConfig
+import com.datadog.gradle.config.junitConfig
+import com.datadog.gradle.config.kotlinConfig
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    // Build
+    id("com.android.library")
+    kotlin("android")
+    id("com.google.devtools.ksp")
+
+    // Publish
+    id("org.jetbrains.dokka")
+
+    // Analysis tools
+    id("com.github.ben-manes.versions")
+
+    // Tests
+    id("de.mobilej.unmock")
+    id("org.jetbrains.kotlinx.kover")
+
+    // Internal Generation
+    id("thirdPartyLicences")
+    id("apiSurface")
+    id("transitiveDependencies")
+    id("binary-compatibility-validator")
+}
+
+android {
+    defaultConfig {
+        consumerProguardFiles("consumer-rules.pro")
+    }
+    compileOptions {
+        isCoreLibraryDesugaringEnabled = true
+    }
+    namespace = "datadog"
```

Comment:
should it be `datadog.trace` at least?

Reply:
yeah I am working on that now...I did not want to change the package in the beginning to be able to update this code easily from the source one. Once I change the package will be a bit harder but I think we need to.

Reply:
I don't think it is related to the package, it is just a namespace afaik, and not necessarily should be the same as package path.